src/typing_manager.ts

```diff
@@ -160,28 +159,27 @@ export class TypingManager implements Disposable {
         await this.client.input(`<c-o>${keys}`);
     };
 
-    private handleCompositeEscapeFirstKey = async (key: string): Promise<void> => {
+    // whenever j or k is pressed,
+    //   if last key pressed was the other (j for k, k for j)
+    //      and pressed recently enough,
+    //   then backspace and simulate esc
+    private handleCompositeEscape = async (key: string): Promise<void> => {
```

Comment:
I suspect I can add another param here for and store it as a private variable if it's not `undefined`. 1 how silly is it to pass a variable on every compositeEscape keypress if you only need it on startup? 2 not sure that's how the args list in `keybindings.json` works.